.github/workflows/deploy.yml

```
@@ -0,0 +1,34 @@
+name: deploy_jenkins
+on: 
+  workflow_dispatch: []
+
```

also we should cancel previously running jobs automatically so if someone hits deploy twice in a row it doesn't mess things up:

```suggestion
concurrency:
  group: deploy_jenkins
  cancel-in-progress: true
```